sonar-ws/src/main/java/org/sonarqube/ws/client/BaseResponse.java

```diff
@@ -40,4 +41,8 @@ public WsResponse failIfNotSuccessful() {
   public boolean hasContent() {
     return code() != HTTP_NO_CONTENT;
   }
+  
+  public void close() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1161)